app/views/dashboard.erb

```diff
@@ -1,13 +1,105 @@
+        <h4 class="panel-title">Robots Average Age</h4>
+      </div>
+      <div id="collapseOne1" class="panel-collapse collapse">
+        <div class="panel-body">
+          <strong><%= @average_age %></strong> years
+        </div>
+      </div>
+    </div>
+
+    <div class="panel panel-default">
+      <div class="panel-heading accordion-toggle collapsed" data-toggle="collapse" data-parent="#accordion1,#accordion2,#accordion3" data-target="#collapseTwo1">
+        <h4 class="panel-title">Robots Hired By Year</h4>
+      </div>
+      <div  id="collapseTwo1" class="panel-collapse collapse">
+        <div class="panel-body">
+          <% @hired_by_year.each do |item| %>
+            <table class="table-condensed">
+              <tr>
+                <td><strong><%= item.first %>:</strong> </td>
+                <td><%= item.last  %></td>
+              </tr>
+            </br>
+          <% end %>
+          </table>
+        </div>
+      </div>
+    </div>
   </div>
+</br>
+
+  <div class="panel-group" id="accordion2">
+    <div class="panel panel-default">
+      <div class="panel-heading accordion-toggle collapsed" data-toggle="collapse" data-parent="#accordion1,#accordion2" data-target="#collapseOne2">
+        <h4 class="panel-title">Robots By Department</h4>
+      </div>
+      <div id="collapseOne2" class="panel-collapse collapse">
+        <div class="panel-body">
+          <% @department_count.each do |item| %>
+            <table class="table-condensed">
+              <tr>
+                <td><strong><%= item.first.capitalize  %></strong></td>
+                <td><%= item.last  %></td>
+              </tr>
+            </br>
+            <% end %>
+          </table>
+        </div>
+      </div>
+    </div>
+
 
+    <div class="panel panel-default">
+      <div class="panel-heading accordion-toggle collapsed" data-toggle="collapse" data-parent="#accordion1,#accordion2,#accordion3" data-target="#collapseTwo2">
+        <h4 class="panel-title">Robots By City</h4>
+      </div>
+      <div id="collapseTwo2" class="panel-collapse collapse">
+        <div class="panel-body">
+          <% @city_count.each do |item| %>
+          <table class="table-condensed">
+            <tr>
+              <td><strong><%= item.first.capitalize  %></td></strong>
+              <td><%= item.last  %></td>
+            </tr>
+          </br>
+          <% end %>
+        </table>
+        </div>
+      </div>
+    </div>
+
+    <div class="panel panel-default">
+      <div class="panel-heading accordion-toggle collapsed" data-toggle="collapse" data-parent="#accordion1,#accordion2,#accordion3" data-target="#collapseThree2">
+        <h4 class="panel-title">Robots By State</h4>
+      </div>
+      <div id="collapseThree2" class="panel-collapse collapse">
+        <div class="panel-body">
+          <% @state_count.each do |item| %>
```

Comment:
you use the work 'item' a lot to describe the thing you are iterating over. Try to use a word that better represents the thing you are iterating over in your collection. Like on this line I think "count" is a better word. Basically you want to make sure someone looking through your code can figure out what a variable holds, or what a method does just by looking at the name. They should be able to easily figure this out without you there to explain it to them.